test/1_eventemitter_embedded_sanity.js

```diff
@@ -10,10 +10,12 @@ describe('1_eventemitter_embedded_sanity', function () {
   var async = require('async');
```

Comment:
These tests are being reformatted later, I did a find replace to remove a
whole bunch of timeout calls, as functional tests should really just have
one call to timeout giving Travis lots of time to pass

On Friday, April 15, 2016, Johan <notifications@github.com> wrote:

> In test/1_eventemitter_embedded_sanity.js
> <https://github.com/happner/happn/pull/66#discussion_r59931049>:
>
> > @@ -10,10 +10,12 @@ describe('1_eventemitter_embedded_sanity', function () {
> >    var async = require('async');
>
> Lots of empty white space added to this file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/happner/happn/pull/66/files/3bf0fa902489a46e3ed9061f834ce34863b2b1f9#r59931049>
>


-- 

SIMON BISHOP

DIRECTOR

+27 (0) 83 282 0703

|     simon@tenacious.digital <brianna@tenacious.digital>

twitter: @_tenacious_ <http://www.twitter.com/_tenacious_>

|     skype: sjbishop

201 Salt Circle Arcade, Salt River, Cape Town, 7925

www.tenacious.digital